packages/project-management-automation/lib/tasks/add-milestone/index.js

```diff
@@ -19,15 +19,20 @@ const DAYS_PER_RELEASE = 14;
  * Returns true if the given error object represents a duplicate entry error, or
  * false otherwise.
  *
- * @param {RequestError} requestError Error to test.
+ * @param {unknown} requestError Error to test.
  *
  * @return {boolean} Whether error is a duplicate validation request error.
  */
 const isDuplicateValidationError = ( requestError ) => {
 	// The included version of RequestError provides no way to access the
 	// full 'errors' array that the github REST API returns. Hopefully they
 	// resolve this soon!
-	return requestError.message.includes( 'already_exists' );
+	const errorMessage = /** @type {undefined | null | {message?: string}} */ ( requestError )
```

Comment:
It looks like it errors in the GitHub action:
https://github.com/WordPress/gutenberg/runs/3482847233

I will try to find a quick fix.

Reply:
https://github.com/WordPress/gutenberg/commit/6730182e37287c871b01897b484c4615cef8a3c6 should fix it.